.drone.yml

```diff
@@ -14,8 +14,11 @@ notify:
 publish:
   catapult:
     url: $$catapult_url
-    application: "marathon-stats"
+    applications:
+      - "marathon-stats-dev"
+      - "marathon-stats-prod"
   docker:
+    docker_version: 1.10.3
```

Comment:
Just added it work around the docker issue we were seeing this morning.  Will remove.